### ISSUE TITLE
Interpolation of images done differently

### ISSUE THREAD

**Issue by wohnjayne** [carpedm20/BEGAN-tensorflow#5]
Just a word of caution: As far as I understand, the way the interpolation of images is implemented here does not correspond to the way it was done in the paper:

In the paper, they approximate the G's  z_1 and  z_2 for two real images using Adam (reversing G(z)), then they interpolate between those z_1, z_2 and show G(z_{interp}).

In this implementation, the discriminator's latent codes  h_1, h_2  are calculated from the two images,  and the discriminator output for the interpolation is shown.

**Comment by carpedm20** [comment]
Yes. I skimmed that part so I didn't noticed that they reversed G. I'll fix this. Thanks!

**Comment by carpedm20** [comment]
I don't know appropriate update step and learning rate to find a z for real image, but at least now the interpolation is generated with G, not D.
